bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/views/rechnung/InvoiceListView.java

```diff
@@ -85,8 +89,12 @@ public class InvoiceListView extends ViewPart implements IRefreshablePart {
 	private InvoiceListBottomComposite invoiceListBottomComposite;
 	private InvoiceListContentProvider invoiceListContentProvider;
 
-	public InvoiceListView() {
+	@Inject
+	@Service(filterExpression = "(" + IModelService.SERVICEMODELNAME + "=ch.elexis.core.model)")
+	private IModelService coreModelService;
 
+	public InvoiceListView() {
+		CoreUiUtil.injectServices(this);
 	}
```

Comment:
Der Aufruf von `CoreUiUtil.injectServices(this)` ist hier nicht nötig, da die Definition der View in _ch.elexis.core.application_ _plugin.xml_ das Attribut `inject="true"` gesetzt hat. Damit sollte das automatisch vom framework erledigt werden.